dbms/src/Flash/FlashService.h

```diff
@@ -134,4 +136,22 @@ class AsyncFlashService final : public FlashService
     }
 };
 
+
```

Comment:
How about move the macro to .cpp file or change its name, it seems not very general while with a general name?

Reply:
ok, I will fix it